### PR TITLE
update_metadata_spec fixes.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Correct tests.

### CODE REVIEW

File: spec/requests/update_metadata_spec.rb [hunk 1]
```diff
@@ -106,8 +106,7 @@
         "identification":#{identification.to_json},
         "structural":{
           "hasMemberOrders":[{"viewingDirection":"right-to-left"}],
-          "isMemberOf":["druid:xx888xx7777"],
-          "hasAgreement":"druid:cd777df7777"
```
Comment (justinlittman) [inline spec/requests/update_metadata_spec.rb:110]: hasAgreement is NOT in the data that is submitted.

File: spec/requests/update_metadata_spec.rb [hunk 2]
```diff
@@ -402,8 +400,11 @@
   end
 
   context 'when tags change' do
+    # In other tests, viewing direction starts at and remains right-to-left.
+    # For this test, starting at left-to-right and changing to right-to-left.
     before do
       allow(AdministrativeTags).to receive(:for).and_return(['Project : Tom Swift', 'Process : Content Type : Book (ltr)'])
+      allow(AdministrativeTags).to receive(:content_type).and_return(['Book (ltr)'], ['Book (rtl)'])
```
Comment (justinlittman) [inline spec/requests/update_metadata_spec.rb:407]: For this test, content type is switching from ltr to rtl.

File: spec/requests/update_metadata_spec.rb [hunk 3]
```diff
@@ -131,7 +130,6 @@
     expect(item).to have_received(:admin_policy_object_id=).with(apo_druid)
 
     # Tags are created.
-    expect(AdministrativeTags).to have_received(:create).with(pid: druid, tags: ['Process : Content Type : Book (rtl)'])
```
Comment (justinlittman) [inline spec/requests/update_metadata_spec.rb:134]: structural isn't changing, so this tag doesn't get created.

File: spec/requests/update_metadata_spec.rb (hunk 1 above)
Comment (justinlittman) [reply spec/requests/update_metadata_spec.rb:110]: That was making it seem like structural was changing, which affects other tests.

**Review by ndushay — APPROVED** [review]

**Pull request merged**